app/presenters/two_factor_auth_code/totpable.rb

```diff
@@ -0,0 +1,18 @@
+module TwoFactorAuthCode
+  module Totpable
+    def auth_app_fallback_link
+      t('links.phone_confirmation.auth_app_fallback_html', link: auth_app_fallback_tag)
+    end
+
+    def empty
+      '.'
+    end
+
+    private
+
+    def auth_app_fallback_tag
+      link_to(t('links.two_factor_authentication.app'),
+              login_two_factor_authenticator_path)
```

Comment:
The Code Climate browser extension is showing that this is not covered by tests.